### ISSUE TITLE
Spotify Support

### ISSUE THREAD

**Issue by FungY911** [sedmelluq/lavaplayer#716]
add support for spotify

**Comment by MCOfficer** [comment]
#155 #519 #600

PS: this repo hasn't seen any work in soon-to-be 2 years, use https://github.com/Walkyst/lavaplayer-fork instead

**Comment by FungY911** [comment]
Sorry for late answer, but this repo (lavaplayer-fork) has like spotify: search and play?

**Comment by MCOfficer** [comment]
no, you still need to write your own solution as mentioned in the other issues.

**Comment by FungY911** [comment]
> no, you still need to write your own solution as mentioned in the other issues.

Thank you, i have it.
